Tests/test_AlignInfo.py

```diff
@@ -139,7 +139,7 @@ def test_pseudo_count(self):
         ic = summary.information_content(e_freq_table=expected,
                                          log_base=math.exp(1),
                                          pseudo_count=1)
-        ic_vector = summary.ic_vector.values()
+        ic_vector = list(summary.ic_vector.values())
```

Comment:
How do you guarantee the list order?
Anyway, I also don't know why summary.ic_vector is a dict; a list makes more sense to me. If you agree, feel free to suggest to change this in Bio.Align.AlignInfo.